index.html

```diff
@@ -758,26 +758,361 @@ <h2>Extensions</h2>
+unambigiously understood, even by software that does not implement
+JSON-LD processing.
+      </p>
+
+      <p>
+The Verifiable Credentials data model also uses a graph-based data model,
+which allows authors to model both simple relationships that describe
+one or more attributes for a single entity or complex multi-entity
+relationships.
+      </p>
+
+      <p>
+The rest of this section describes how to author extensions that build on
+top of the Verifiable Credentials Data Model.
       </p>
 
       <section>
         <h3>Creating New Credential Types</h3>
 
-        <ul>
+        <p>
+We expect the most common extensions to the Verifiable Credentials Data Model
+to be new credential types. Whenever someone has something to say about one or
+more entities and they want their authorship to be verifiable, they should use
+a Verifiable Credential. Sometimes there may be an existing credential type,
+that someone else has created, that can be reused to make the statements they
+want to make. However, there are often cases where new credential types are
+needed.
+        </p>
+
+        <p>
+New credential types can be created by following a few steps. This guide will
+also walk you through creating an example new credential type. At a high level,
+the steps to follow are:
+        </p>
+
+        <ol>
           <li>
-Design the Data Model.
+Design the data model.
           </li>
           <li>
-Create the JSON-LD Context.
+Create a new JSON-LD context.
           </li>
           <li>
 Select a publishing location.
           </li>
           <li>
-Use the new JSON-LD Context.
+Use the new JSON-LD context when issuing new credentials.
           </li>
-        </ul>
+        </ol>
+
+        <p>
+So, let's walk through creating a new credential type which we will
+call <code>ExampleAddressCredential</code>. The purpose of this credential
+will be to express a person's postal address.
+        </p>
+
+        <h4>Design the data model</h4>
+
+        <p>
+First, we must design a data model for our new credential type. We know that
+we will need to be able to express the basics of a postal address, things
+like a person's city, state, and zipcode. Of course, those items are quite
+US centric, so we should consider internationalizing those terms. But before
+we go further, since we're using [LINKED-DATA] vocabularies, there is a good
+chance that commonly known concepts may already have a vocabulary that someone
+else has created that we can leverage.
+        </p>
+
+        <p>
+If we are going to use someone else's vocabulary, we will want to make sure it
+is stable and unlikely to change in any significant way. There may even be
+technologies that we can make use of that store immutable vocabularies that
+we can reference, but those are not the focus of this example. Here we will
+rely on the inertia that comes from a very popularly used vocabulary on the
+Web, schema.org. It turns out that this vocabulary has just what we need; it
```

Comment:
schema.org is popular, and much of it is fairly stable at the moment, but there is no guarantee or assurance that it will remain so, and I strongly advise against using it as an example of an *immutable* ontology/vocabulary.

Reply:
The clause before we talk about schema.org says "[immutable vocabularies] are not the focus of this example." So I was trying to mention that immutable vocabularies are possible, but that's not what we're using here... we're using schema.org, a vocabulary that is *not* immutable. Do you think I need to change any of the language here to make that more clear?

Reply:
The issue I see is that we're telling people to rely on the "inertia" of a "stable vocabulary" so that future evaluators of the statements in their VC will understand what they meant by a given attribute -- but there's no way to assure that the meaning of attribute "schema:example" will be the same at that evaluation time, nor any way to check whether it has changed, nor any way to see what such a change was (insignificant change in punctuation of the description? complete reversal of meaning of the attribute? something else?).

It seems to me that these potential downfalls of a shared and evolving vocabulary/ontology ought to be brightly highlighted, not glossed over as if they don't exist at all.